docs/integrations/destinations/weaviate-migrations.md

```diff
@@ -0,0 +1,18 @@
+# Weaviate Migration Guide
+
+## Upgrading to 0.2.0
+
+This version adds several new features like flexible embedding options, overwrite and append+dedup sync modes , but also contains multiple changes that require a migration of existing connections. This is done to align the behavior of vector database destinations in Airbyte. The following changes are included:
+
+### Changed configuration object structure
+
+Due to a change of the configuration structure, it's necessary to reconfigure existing destinations with the same information (e.g. credentials).
+
+### Auto-generated ids
+
+It's not possible anymore to configure id fields in the destination. Instead, the destination will generate a UUID for each Weaviate object. The id per record is stored in the `_ab_record_id` property and can be used to identify Weaviate objects by Airbyte record.
```

Comment:
I'm not 100% sure on this, but I think allcaps "ID" is proper for generic references to the "ID" identifier.

```suggestion
It's no longer possible to configure ID fields in the destination. Instead, the destination will generate a UUID for each Weaviate object. The ID per record is stored in the `_ab_record_id` property and can be used to identify Weaviate objects by Airbyte record.
```

Or, if `id` is a field name:

```suggestion
It's no longer possible to configure `id` fields in the destination. Instead, the destination will generate a UUID for each Weaviate object. The `id` for each record is stored in the `_ab_record_id` property and can be used to identify Weaviate objects by Airbyte record.
```